03_python_fastapi_project/main.py

```
@@ -111,6 +112,126 @@ async def delete_product(product_id: int, db: AsyncSession = Depends(get_db)):
     return {"message": "Product deleted successfully"}
 
 
+@app.get("/basket/", response_model=List[BasketItemDTO])
+async def get_basket(db: AsyncSession = Depends(get_db)):
```

Ruff (0.13.3)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)